### DISCUSSION TITLE
Sling icon

### DISCUSSION THREAD

**Discussion by pranavkrishnasuresh** [elrumo/macOS_Big_Sur_icons_replacements#1247] [Icon request]
Can someone make a sling icon? please center this in a white square.

![](https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Sling_TV_logo.svg/1200px-Sling_TV_logo.svg.png)

**Comment by OMGzilla-git** [comment]
Done. It's in my profile while awaiting approval.

https://macosicons.com/u/OMGzilla

EDIT: I overlooked that you wanted a white background (the official iOS one has a black background). I re-did it, so now there are one of each available.
